changelog/unreleased/enhancement-redesign-status-indicators

```diff
@@ -0,0 +1,8 @@
+Enhancement: Redesign OcStatusIndicators
+
+We've redesigned the share/status indicators to fit
```

Comment:
In theory we did redesign them to fit the new UI better, but that doesn't matter too much 😉